### PR TITLE
feat: add email to customer.

### PR DESCRIPTION
**Pull request by shidenkai0**

### CODE REVIEW

File: pkg/api/gen/api.gen.go [hunk 1]
```diff
@@ -31,29 +35,37 @@ type Car struct {
 	Year     int    `json:"year"`
 }
 
+// CreateCustomerRequest defines model for CreateCustomerRequest.
+type CreateCustomerRequest struct {
+	Email openapi_types.Email `json:"email"`
```
Comment (alexisdcl) [inline pkg/api/gen/api.gen.go:40]: Why openapi_types ?

File: go.mod
Comment (jcruz97) [inline go.mod:1]: Are repositories update a result of a go mod tidy call ?

File: pkg/api/gen/api.gen.go (hunk 1 above)
Comment (VaneckQuentin) [reply pkg/api/gen/api.gen.go:40]: This is auto generated via openapi by specifying the format in the property in rental.yml

File: pkg/api/api.go [hunk 2]
```diff
@@ -230,18 +232,34 @@ func (s *Server) GetCustomerById(ctx echo.Context, customerId int64) error {
...
-	if err := ctx.Bind(&CreateCustomer); err != nil {
+	updateCustomer := gen.UpdateCustomerRequest{}
+	if err := ctx.Bind(&updateCustomer); err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err.Error())
 	}
-	customer := rental.Customer{ID: int(customerId), Name: CreateCustomer.Name}
-	err := s.CustomerCRUDService.Update(customer)
+
+	// Get the existing customer data
+	existingCustomer, err := s.CustomerCRUDService.Get(int(customerId))
```
Comment (alexisdcl) [inline pkg/api/api.go:241]: Do we really need to cast the paramater int 64 to int ?

**Pull request merged**